app/hallebarde/infrastructure/token_repository.py

```diff
@@ -0,0 +1,14 @@
+import boto3
+import hallebarde.config
+
+client = boto3.resource('dynamodb')
+table = client.Table(f'hallebarde-{hallebarde.config.ENVIRONMENT}-table')
+
+
+def save(id: str, upload_token: str, download_token: str):
```

Comment:
I would name the parameter "id" differently as id already refers to a built-in function in python 
--> https://docs.python.org/fr/3.6/library/functions.html#id

Reply:
Would you be ok with 'identifier' ?